Yoast/wordpress-seo
# Load correct css files

## Context


* Some files on the feature-branch were still sass files. This PR creates CSS files and fixes the grunt build to correctly load the CSS files. There's still one 'to do' in the code, this can be removed together with #14298 

## Summary


This PR can be summarized in the following changelog entry:

* Fixes Grunt Build to load CSS files correctly
* Removes last traces of SASS

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* This PR needs to be tested together with https://github.com/Yoast/javascript/pull/640
* Checkout the current state of the feature-branch and see that some CSS files are not loaded correctly, causing unwanted changes in the admin-bar and on the post-edit page.
* Checkout both branches
* Run a grunt build
* Check the pages again, the CSS should now be loaded correctly. 
* The social and advanced tabs on the post-edit page are still missing some styling, Reactification team is working on these components.

## UI changes

## Documentation

## Quality assurance


Fixes #
